app/controllers/api/functions.php

```diff
@@ -442,6 +451,57 @@
         ]), Response::MODEL_RUNTIME_LIST);
     });
 
+App::get('/v1/functions/specifications')
+    ->groups(['api', 'functions'])
+    ->desc('Get available function runtime specifications')
+    ->label('scope', 'functions.read')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
```

Comment:
I think this can be public endpoint.
If not agreed upon, then at least `APP_AUTH_TYPE_ADMIN` alongside API key

Reply:
Added `APP_AUTH_TYPE_ADMIN`, I used only APP_AUTH_TYPE_KEY as that's what we do in the listRuntimes endpoint. Do we want to consider modifying that?